src/main/java/org/jenkinsci/plugins/workinghours/ValidationResult.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.plugins.workinghours;
+
+import net.sf.json.JSONObject;
+
+public class ValidationResult {
+    public boolean isValid;
+    public String errorMessage;
+    public String fieldName;
+
+    public ValidationResult(boolean isValid, String field, String errorMessage) {
+        this.isValid = isValid;
+        this.fieldName = field;
+        this.errorMessage = errorMessage;
```

Comment:
Is this the only places these fields are ever set? If so, they should probably be marker final.

Reply:
Good Idea 💡